### DISCUSSION TITLE
Domain is pending verification

### DISCUSSION THREAD

**Discussion by epleaner** [vercel/community#7384] [Account Issue]
### Page to Investigate

<project_url>settings/domains

### Steps to Reproduce

I am attempting to move a domain from Netlify to Vercel. I have deleted the project on Netlify and set one up in Vercel. When trying to add my domain, I see: 

Domain is pending verification

Another Vercel account is using this domain. Set the following TXT record on _vercel.<domain>.com to use <domain>.com in this project. If you wish to [transfer](https://vercel.com/guides/how-can-i-move-a-domain-to-a-team#move-button) the entire domain, this will need to be done by the current domain owner. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.

I have no idea which other Vercel account would be using this domain.

**Comment by pawlean** [comment]
Hi @epleaner!

Do you have access to your domain registrar where you first got your domain? 

Once you get access, you need to follow the instructions:

> Set the following TXT record on _vercel..com to use .com in this project. If you wish to [transfer](https://vercel.com/guides/how-can-i-move-a-domain-to-a-team#move-button) the entire domain, this will need to be done by the current domain owner. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.

**Reply by AlbertGithubHome** [reply]
I have access to my domain registrar, but I have encountered this domain name verification problem, need to get some help, the domain name verification needs to add the name of `_vercel` TXT record, but the domain name service provider we use JPdirect does not allow to add records containing the underline `"_"`, how to solve it? Are there other ways to verify the ownership of a domain name?

**Reply by pawlean** [reply]
@AlbertGithubHome Saw your post in the new community, let's keep the conversation there.

**Comment by amyegan** [comment]
Closing because the conversation moved to https://vercel.community/t/having-trouble-verifying-my-domain-name-with-txt-record-vercel/194/2
